Monika After Story/game/script-topics.rpy

```diff
@@ -2353,7 +2353,21 @@ label monika_rain_holdme:
 
         call monika_holdme_end
         # small affection increase so people don't farm affection with this one.
-        $ mas_gainAffection(modifier=0.25)
+        python:
+            # Monika likes being held for long periods of time, the longer the better
+            if elapsed_time <= datetime.timedelta(minutes=10):
+                mod = 0.25
+            elif ((elapsed_time > datetime.timedelta(minutes=10)) and (elapsed_time <= datetime.timedelta(minutes=30))):
```

Comment:
```suggestion
            elif elapsed_time <= datetime.timedelta(minutes=30):
```

this is what I meant, you don't need the 10 minute check since the last check covered this. you can do this for all subsequent checks.

Reply:
Ohhhhhh. Thank you for the clarification. I'll do that.